### PR TITLE
chore(log): add  info logs for http handlers.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

## Changelog

- Improvement

## Related Issues

Fixes #issue

### CODE REVIEW

File: query/src/servers/http/clickhouse_handler.rs [hunk 1]
```diff
@@ -306,6 +306,11 @@ pub async fn clickhouse_handler_post(
     let default_format = get_default_format(&params, headers).map_err(BadRequest)?;
     let mut sql = params.query();
     sql.push_str(body.into_string().await?.as_str());
+    // other parts of the request already logged in middleware
+    tracing::info!(
+        "receive clickhouse http post, (query + body)[0..100] = {}",
+        &sql[0..100]
```
Comment (Xuanwo) [inline query/src/servers/http/clickhouse_handler.rs:312]: What happens if the user inputs a SQL with only 10 bytes?

Comment (youngsofun) [reply query/src/servers/http/clickhouse_handler.rs]: my fault, fixed,   thought it is python..

File: query/src/servers/http/clickhouse_handler.rs [hunk 2]
```diff
@@ -306,6 +306,14 @@ pub async fn clickhouse_handler_post(
     let default_format = get_default_format(&params, headers).map_err(BadRequest)?;
     let mut sql = params.query();
     sql.push_str(body.into_string().await?.as_str());
+    let N = 100;
+    // other parts of the request already logged in middleware
+    let msg = if sql.len() > N {
+        sql[0..N].to_string() + &format!("...(omit {} bytes)", sql.len() - N)
```
Comment (Xuanwo) [inline query/src/servers/http/clickhouse_handler.rs:312]:
```suggestion
        format!("{}...(omit {} bytes)", sql[0..N], sql.len() - N)
```

**Review by Xuanwo — APPROVED** [review]

File: query/src/servers/http/middleware.rs [hunk 3]
```diff
@@ -136,7 +136,8 @@ impl<E: Endpoint> Endpoint for HTTPSessionEndpoint<E> {
     type Output = E::Output;
 
     async fn call(&self, mut req: Request) -> PoemResult<Self::Output> {
-        tracing::debug!("receive http request: {:?},", req);
+        // method, url, version, header
+        tracing::info!("receive http handler request: {:?},", req);
```
Comment (Xuanwo) [inline query/src/servers/http/middleware.rs:140]:
```suggestion
        tracing::info!("receive http handler request: {req:?}");
```

**Review by PsiACE — APPROVED** [review]

**Pull request merged**